src/packager.ts

```diff
@@ -10,54 +10,57 @@ import {
   paginateListObjectsV2,
 } from "@aws-sdk/client-s3";
 import { Upload } from "@aws-sdk/lib-storage";
+import { AWSLambda as Sentry } from "@sentry/serverless";
 import { ScheduledHandler } from "aws-lambda";
 import { create as struct } from "superstruct";
 import { create as createTar } from "tar";
 import { localFromISO } from "./datetime";
+import { exception } from "./errors";
 import { Environment } from "./packager/structs";
 
-class BadS3ResponseError extends Error {}
-class OutputKeyExistsError extends Error {}
-
-export const handler: ScheduledHandler = async ({ time }) => {
-  // Copy the env to avoid `optional` properties becoming the string "undefined"
-  // instead of real `undefined`; this may have to do with the automatic string
-  // conversion behavior of `process.env`
-  const env = struct({ ...process.env }, Environment);
-  const client = makeS3Client(env);
-  const {
-    S3_BUCKET: bucket,
-    S3_PREFIX_OUTPUT: outputPrefix,
-    S3_PREFIX_SOURCE: sourcePrefix,
-  } = env;
-
-  const serviceDay = localFromISO(time).minus({ days: 1 }).toISODate();
-  const shortServiceDay = serviceDay.replace(/-/g, "");
-  const outputKey = path.posix.join(outputPrefix, `${shortServiceDay}.tar.gz`);
-
-  if (await objectExists(client, bucket, outputKey))
-    throw new OutputKeyExistsError(outputKey);
-
-  const archiveRoot = await concatAllObjects(
-    client,
-    bucket,
-    path.posix.join(sourcePrefix, serviceDay),
-    // Mimic the directory structure of the old OCS.LogUploader from RTR
-    path.join("root", "persistent-state", `${shortServiceDay}.txt`)
-  );
-
-  const upload = new Upload({
-    client,
-    params: {
-      Body: createTarStream(archiveRoot, ["."]),
-      Bucket: bucket,
-      ContentType: "application/gzip",
-      Key: outputKey,
-    },
-  });
-
-  await upload.done();
-};
+Sentry.init();
+
+export const handler: ScheduledHandler = Sentry.wrapHandler(
+  async ({ time }) => {
+    // Copy `process.env` to avoid `optional` properties becoming the string
+    // "undefined" instead of real `undefined`; this may have to do with its
+    // automatic string conversion behavior
+    const env = struct({ ...process.env }, Environment);
+    const client = makeS3Client(env);
+    const {
+      S3_BUCKET: bucket,
+      S3_PREFIX_OUTPUT: outputPrefix,
+      S3_PREFIX_SOURCE: sourcePrefix,
+    } = env;
+
+    const serviceDay = localFromISO(time).minus({ days: 1 }).toISODate();
+    const outputLabel = serviceDay.replace(/-/g, "");
```

Comment:
question: does it make sense to separate the reformatting out into a separate commit here? Not for review purposes, but for history?

Reply:
If by "reformatting" you mean the whole re-indentation of this function... I'm not sure? When the `wrapHandler` call is present, Prettier formats it like this; when it isn't, Prettier formats it the other way. I suppose we could throw in a temporary `// prettier-ignore`, then remove it in the subsequent commit. There is always the whitespace-ignoring diff mode, though.

Reply:
Oh, ok. I didn't realize that the indentation level actually increased.